bundles/thunk_bundle.js

```diff
@@ -7001,7 +7010,7 @@ module.exports = (() => {
   }
   __name(handleError, "handleError");
   function handleFetcherStatusError(fetchResult, fetchRequest) {
-    if (fetchResult.status >= 300) {
+    if (fetchResult.status >= 400) {
```

Comment:
This used to mimic request-promise behavior and throw for anything > 300. This seems undesirable, a user is likely expecting to get a response to expect and not an exception (which they can still inspect to get the same data, but seems gross). This never came up before because the SDK would never let you get 301/302 responses until now.